TF/src/TFParser.py

```diff
@@ -202,6 +203,15 @@ def parse_graph(self, file_path, model_name, category, is_saved_model, input_ope
             adj_list = new_adj_list
             del new_adj_list
 
+            # for index, node in enumerate(nodes):
+            #     if node.operator_type == "Reshape":
+            #         if index in adj_list:
+            #             print(adj_list[index])
+            #             for _, node_index in adj_list[index]:
+            #                 print(nodes[node_index].operator_type)
+            #                 if nodes[node_index].operator_type == "MatMul":
+            #                     print("Got One!")
+
```

Comment:
May be remove this if this is not needed.

Reply:
Fixed in latest commit

Reply:
Already fixed in a previous commit